CHANGELOG.md

```diff
@@ -0,0 +1,140 @@
+
+## Release Notes - Identity Service
+
+### Version: <em>identity-service-1.0</em>
+
+<h2>Epic
+</h2>
+<ul>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-134'>AUTH-134</a>] -         Identity Service 1.0
+</li>
+</ul>
+
+<h2>Story
+</h2>
+<ul>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-65'>AUTH-65</a>] -         Integrate code into APS to implement the use of the internal auth token
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-130'>AUTH-130</a>] -         Make Alfresco-Identity-Service Github repo public
```

Comment:
Although the end result of this story is user visible, it seems to me the users don't need this level of granularity in Release Notes.

Reply:
I kept that one in just because it makes it clear this is intended to be open sourced, talks about licensing and contribution process.